README.md

```diff
@@ -35,9 +35,109 @@ Follow these steps to delete a dataspace instance and free up the corresponding
+development purposes with the following exception to use the profile `ui` as described below.
+
+You need to check out the
+repository [eclipse-dataspaceconnector/DataDashboard](https://github.com/eclipse-dataspaceconnector/DataDashboard) or
+your corresponding fork. Set the environment variable `MVD_UI_PATH` to the path of the DataDashboard repository.
+
+Bash:
+
+```bash
+export MVD_UI_PATH="/path/to/mvd/datadashboard"
+docker-compose --profile ui -f system-tests/docker-compose.yml up --build
+```
+
+PowerShell:
+
+> Docker Compose expects the path to use forward slashes instead of backslashes.
+
+```powershell
+$Env:MVD_UI_PATH="/path/to/mvd/datadashboard"
+docker-compose --profile ui -f system-tests/docker-compose.yml up --build
+```
+
+The profile `ui` creates three Data Dashboards each connected to an EDC participant. The respective `app.config.json`
+files can be found in the respective directories:
+
+- `resources/appconfig/provider/app.config.json`
+- `resources/appconfig/consumer-eu/app.config.json`
+- `resources/appconfig/consumer-us/app.config.json`
+
+That's it to run the local development environment. The following section `Run A Standard Scenario Locally` describes a
+standard scenario which can be optionally used with the local development environment.
+
+> Tip: The console output from the services spun up by Docker compose can be noisy. To decrease the output from the
+> services on the console set `EDC_CATALOG_CACHE_EXECUTION_PERIOD_SECONDS` to a higher value, e.g. 60, for each EDC
+> participant in `system-tests/docker-compose.yml`.
+
+### Run A Standard Scenario Locally
+
+Prerequisite: create a test document manually:
+
+- Connect to the **local** blob storage account (provided by Azurite) of the provider.
+  - Storage account name: `providerassets`, storage account key: `key1`.
+  - [Microsoft Azure Storage Explorer](https://azure.microsoft.com/features/storage-explorer/) can be used to connect to the local
+    storage account on `localhost:10000`.
+- Create a container named `src-container`. (Container name is defined for Postman request `Publish Master Data`
+  in `deployment/data/MVD.postman_collection.json`)
+- Copy `deployment/terraform/participant/sample-data/text-document.txt` into the newly created container.
+  - N.B.: it does not have to be this exact file as long you create a file which has the name `text-document.txt`.
+
+All this can also be done using Azure CLI with the following lines from the root of the MVD repository:
+
+Bash:
+
+```bash
+conn_str="DefaultEndpointsProtocol=http;AccountName=providerassets;AccountKey=key1;BlobEndpoint=http://127.0.0.1:10000/providerassets;"
+az storage container create --name src-container --connection-string $conn_str
+az storage blob upload -f .\deployment\terraform\participant\sample-data\text-document.txt --container-name src-container --name text-document.txt --connection-string $conn_str
```

Comment:
For bash you should use slashes '/' instead of backslashes

Reply:
Good catch! Fixed this.